Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,71 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "Blob"
+    ],
+    "name": "BlobTrigger",
+    "identity": "Azure.Function.CSharp.Isolated.BlobTrigger.2.x",
```

Comment:
Change to 3.x

Reply:
Let's leave this to `2.x` to match what we do in .NET Core 3? When we update there, we can update these too.

Reply:
Can't change to 3.x in .NET Core 3, if we do that VS would throw null reference exception. We should change these while we have the chance :)

Reply:
Fair! I changed them.